### PR TITLE
added relations to comment.models.py

### PR DESCRIPTION
**Pull request by skramar**
Added Team relation
Added Event relation
Added Vote relation
Added CustomUser relation

## Code reviewers

- [x] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [x] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/comment/models.py [hunk 1]
```diff
@@ -2,29 +2,49 @@
...
 
     """
 
     text = models.CharField(max_length=1024)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    team = models.ForeignKey(Team, on_delete=models.CASCADE)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE)
+    vote = models.ForeignKey(Vote, null=True, on_delete=models.CASCADE)
+    author = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
```
Comment (makarman) [inline eventually/comment/models.py]: remove on_delete=models.CASCADE in author

File: eventually/comment/models.py [hunk 2]
```diff
@@ -78,18 +121,35 @@ def get_by_id(comment_id):
...
 
-        :param text: - text of comment
+        :param team: Certain's Team's object. Is required.
+        :type team: Team Model
+
+        :param event: Certain's Event's object. Is required.
+        :type event: Event Model
+
+        :param author: Certain's CustomUser's object. Is required.
+        :type author: CustomUser Model
```
Comment (makarman) [inline eventually/comment/models.py]: on model but object

File: eventually/comment/models.py [hunk 3]
```diff
@@ -126,3 +186,9 @@ def delete_by_id(comment_id):
             return True
         except (Comment.DoesNotExist, AttributeError):
             pass
+
+    @staticmethod
+    def get_all(start=0, end=10):
+        """Get first 10 Comment objects ordered by id"""
```
Comment (makarman) [inline eventually/comment/models.py]: delete this method

File: eventually/comment/models.py (hunk 1 above)
Comment (skramar) [reply eventually/comment/models.py]: done

File: eventually/comment/models.py (hunk 2 above)
Comment (skramar) [reply eventually/comment/models.py]: done

File: eventually/comment/models.py (hunk 3 above)
Comment (skramar) [reply eventually/comment/models.py]: done

**Review by ssOleg — DISMISSED** [review]

**Review by redmyt — APPROVED** [review]

**Pull request merged**